UnitTests/Views/ViewDisposalTest.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Reflection;
+using Terminal.Gui;
+using Xunit;
+using Console = Terminal.Gui.FakeConsole;
+using Xunit.Sdk;
+using Microsoft.VisualStudio.TestPlatform.Utilities;
+using Xunit.Abstractions;
+using System.Threading;
+using Terminal.Gui.Graphs;
+
+namespace UnitTests.ViewsTests {
+	public class ViewDisposalTest {
+		ITestOutputHelper output;
+		public ViewDisposalTest (ITestOutputHelper output)
+		{
+			{
+				this.output = output;
+			}
+		}
+#nullable enable
+		Dictionary<Type, object? []?> special_params = new Dictionary<Type, object? []?> ();
+#nullable restore
+		[Fact]
+		[AutoInitShutdown]
+		public void TestViewsDisposeCorrectly ()
+		{
+			var reference = DoTest ();
+			for (var i = 0; i < 10 && reference.IsAlive; i++) {
+				GC.Collect ();
+				GC.WaitForPendingFinalizers ();
+			}
+
+			if (reference.IsAlive) {
+				Assert.True (((View)reference.Target).WasDisposed);
+				Assert.Fail ($"Some Views didnt get Garbage Collected: {((View)reference.Target).Subviews}");
+			}
+		}
+		void getSpecialParams ()
+		{
+			special_params.Clear ();
+			//special_params.Add (typeof (LineView), new object [] { Orientation.Horizontal });
+		}
+		WeakReference DoTest ()
+		{
+			getSpecialParams ();
+			View Container = new View ();
+			Toplevel top = Application.Top;
+			var views = GetViews ();
+			foreach (var view in views) {
+				View instance;
+				//Create instance of view and add to container
+				if (special_params.ContainsKey (view)) {
+					instance = (View)Activator.CreateInstance (view, special_params [view]);
+				}
+				else
+					instance = (View)Activator.CreateInstance (view);
```

Comment:
Please add `{` and `}` (we try to be very consistent about always using braces for nested statements). Thanks.